ohsome_quality_api/topics/presets.yaml

```diff
@@ -164,6 +164,36 @@ cycleway:
     - core
     - bkg
 
+power_lines:
+  name: power_lines
+  description: >-
+      All linear OSM features referring to power lines.
+  endpoint: elements
+  aggregation_type: length
+  filter: >-
+    power=lines and geometry:line
+  indicators:
+    - attribute-completeness
+    - mapping-saturation
+    - currentness
+  projects:
+    - ohmygrid
+
+power_substation:
+  name: power_substation
+  description: >-
+      A facility which controls the flow of electricity in a power network with transformers, switchgear or compensators
+  endpoint: elements
+  aggregation_type: count
+  filter: >-
+    power=substation and geometry:point
```

Comment:
I think often this is also mapped as a way / polygon: https://taginfo.openstreetmap.org/tags/power=substation. 
I think you could just remove the geometry filter.